plenary/plenary-14.yaml

```diff
@@ -0,0 +1,326 @@
+
+
+  - dates:
+      - 1998-03-12
+      - 1998-03-13
+    identifier: 145
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: resolves
+        date_effective:
+          start: 1998-03-12
+          end: 1998-03-13
+        message: |
+        ISO/TC 154 resolves that a recommendation should be developed by the TC154 secretariat for the ISO/TMB proposing the setting up of a single ISO technical committee covering the meta standards and the implementation level standards in support of EDI/EC. This would group the existing ISO/TC154 with ex JTC1/SC14 and SC30. It would have the impact of streamlining the standardization process, including positive cost and resource impacts, as well as providing a single focal point for ISO on EDI/EC matters in the ISO/IEC/UN-ECE MoU.
+        ISO/TC 154 notes that this recommendation is in line with JTC1/SC30 resolution 1997/1
+
+
+  - dates:
+      - 1998-03-12
+      - 1998-03-13
+    identifier: 146
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: resolves
+        date_effective:
+          start: 1998-03-12
+          end: 1998-03-13
+        message: ISO/TC 154 having noted the ISO/TC 154 US TAG  position with regard to the extended MoU,and in particular, the sentence « We are aware that the ISO/TMB also agreed with not extending the MoU to include CALS » have concluded that their interpretation is incorrect. ISO/TC 154 accordingly resolves that ISO/CS responds officially to the US TAG clearly setting out the correct TMB position
+
+
+  - dates:
+      - 1998-03-12
+      - 1998-03-13
+    identifier: 147
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: supports
+        date_effective:
+          start: 1998-03-12
+          end: 1998-03-13
+        message: ISO/TC 154 supports the extension to the MoU (MoU/CC/97 N0022 Rev1) in line with resolution  94/1997 of the ISO/TMB
+      - type: requests
+        date_effective:
+          start: 1998-03-12
+          end: 1998-03-13
+        message: requests its members to provide comments on the draft extended MoU to the TC/154 secretariat by the 9th of April 1998
+
+
+  - dates:
+      - 1998-03-12
+      - 1998-03-13
+    identifier: 148
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: nominates
+        date_effective:
+          start: 1998-03-12
+          end: 1998-03-13
+        message: ISO/TC 154 nominates Alain Thienot and Denis Hill to represent ISO/TC 154 at the Miami MoU/CC meeting from the 29th April to the 1st of May 1998
```

Comment:
Here, I used the word `nominates` as action type. Is it correct?

Reply:
Yes.